src/workerd/server/server.c++

```diff
@@ -1526,7 +1526,7 @@ kj::Own<Server::Service> Server::makeWorker(kj::StringPtr name, config::Worker::
         .softLimit = 16ull << 20,
         .hardLimit = 128ull << 20,
         .staleTimeout = 30 * kj::SECONDS,
-        .dirtyKeySoftLimit = 64,
+        .dirtyListByteLimit = 2ull << 20,
```

Comment:
nit: This is the dumbest thing but could I convince you to do something like this:
```
.dirtyListByteLimit = 2*(1ull << 20), // 2MiB
```